doc/languages-frameworks/go.xml

```diff
@@ -94,7 +94,7 @@ deis = buildGoPackage rec {
 
   goDeps = ./deps.nix; <co xml:id='ex-buildGoPackage-3' />
 
-  buildFlags = "--tags release"; <co xml:id='ex-buildGoPackage-4' />
+  buildFlags = [ "--tags" "release" ]; <co xml:id='ex-buildGoPackage-4' />
```

Comment:
not ok

Reply:
Can you expand on what this issue is here?

Reply:
oh, sorry, it is fine, I misread it.